config.yml

```
@@ -28,3 +28,117 @@ paths:
+    # Specific API Endpoints
+    - "api\\.example\\.com/tracking"
+    - "/api/candidatures/anticheating/"
+    - "sentry_key="
+
+# Browser Configuration
+browser:
+  # Viewport settings
+  viewport:
+    width: 1920
+    height: 1080
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)